packages/bundles/src/all-imports.js

```diff
@@ -0,0 +1,90 @@
+/* Core */
+import '@vaadin/accordion';
+import '@vaadin/app-layout';
+import '@vaadin/app-layout/vaadin-drawer-toggle.js';
+import '@vaadin/avatar';
+import '@vaadin/avatar-group';
+import '@vaadin/button';
+import '@vaadin/checkbox';
+import '@vaadin/checkbox-group';
+import '@vaadin/combo-box';
+import '@vaadin/context-menu';
+import '@vaadin/custom-field';
+import '@vaadin/date-picker';
+import '@vaadin/date-time-picker';
+import '@vaadin/details';
+import '@vaadin/dialog';
+import '@vaadin/email-field';
+import '@vaadin/form-layout';
+import '@vaadin/form-layout/vaadin-form-item.js';
+import '@vaadin/grid';
+import '@vaadin/grid/vaadin-grid-column-group.js';
+import '@vaadin/grid/vaadin-grid-column.js';
+import '@vaadin/grid/vaadin-grid-filter.js';
+import '@vaadin/grid/vaadin-grid-selection-column.js';
+import '@vaadin/grid/vaadin-grid-sorter.js';
+import '@vaadin/grid/vaadin-grid-tree-toggle.js';
+import '@vaadin/horizontal-layout';
+import '@vaadin/icon';
+import '@vaadin/icons';
+import '@vaadin/integer-field';
+import '@vaadin/item';
+import '@vaadin/list-box';
+import '@vaadin/login';
+import '@vaadin/login/vaadin-login-overlay.js';
+import '@vaadin/login/vaadin-login-form.js';
+import '@vaadin/menu-bar';
+import '@vaadin/message-input';
+import '@vaadin/message-list';
+import '@vaadin/notification';
+import '@vaadin/number-field';
+import '@vaadin/password-field';
+import '@vaadin/progress-bar';
+import '@vaadin/radio-group';
+import '@vaadin/scroller';
+import '@vaadin/select';
+import '@vaadin/split-layout';
+import '@vaadin/tabs';
+import '@vaadin/text-area';
+import '@vaadin/text-field';
+import '@vaadin/time-picker';
+import '@vaadin/upload';
+import '@vaadin/vertical-layout';
+import '@vaadin/virtual-list';
+/* Pro */
+import '@vaadin/board';
+import '@vaadin/charts';
+import '@vaadin/confirm-dialog';
+import '@vaadin/cookie-consent';
+import '@vaadin/crud';
+import '@vaadin/grid-pro';
+import '@vaadin/grid-pro/vaadin-grid-pro-edit-column.js';
+import '@vaadin/rich-text-editor';
+/* Theme */
+import '@vaadin/vaadin-lumo-styles';
+/* Internal dependencies */
+import '@vaadin/component-base';
+import '@vaadin/field-base';
+import '@vaadin/input-container';
+// ignore non-resolvable import '@vaadin/vaadin-license-checker';
+import '@vaadin/vaadin-list-mixin';
+import '@vaadin/vaadin-themable-mixin';
+import '@vaadin/vaadin-development-mode-detector';
+import '@vaadin/vaadin-usage-statistics';
+// FIXME: make components not depend on those
+// ignore deprecated import '@vaadin/vaadin-overlay';
+// ignore deprecated import '@vaadin/vaadin-context-menu';
+/* External dependencies */
+import '@lit/reactive-element';
+// ignore non-resolvable import '@polymer/iron-flex-layout';
+import '@polymer/iron-icon';
+import '@polymer/iron-iconset-svg';
+import '@polymer/iron-media-query';
+import '@polymer/iron-meta';
+import '@polymer/polymer';
+import '@webcomponents/shadycss';
+import 'cookieconsent';
```

Comment:
What's the point behind importing these transitive dependencies? They are already imported by components.

Also, in case of `highcharts`, we import explicit files, not just the default entrypoint.